test/integration/targets/azure_rm_sqldatabase/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+- name: Create SQL Server
+  azure_rm_sqlserver:
+    resource_group: "{{ resource_group }}"
+    name: sql-test-server-dauih
+    location: eastus
+    admin_username: mylogin
+    admin_password: Testpasswordxyz12!
+- name: Create instance of SQL Database
+  azure_rm_sqldatabase:
+    resource_group: "{{ resource_group }}"
+    server_name: sql-test-server-dauih
+    name: test-database
+    location: eastus
+  register: output
+- name: Assert the resource instance is well created
+  assert:
+    that:
+      - output.changed
+      - output.status == 'Online'
```

Comment:
Would be good to check that some of the return values like `id` is defined, like so

```
- name: Assert the resource instance is created
  assert:
  that:
  - output.changed
  - output.status == 'Online'
  - output.id is defined
  - output.database_id is defined
```